src/UI/class-settings-page.php

```diff
@@ -955,7 +956,8 @@ private function validate_basic_section( $input ) {
 
 			$input['api_secret'] = $this->get_unobfuscated_value( $input['api_secret'], $this->parsely->get_api_secret() );
 			$api_secret_length   = strlen( $input['api_secret'] );
-			if ( $api_secret_length > 0 && $api_secret_length <= 30 ) {
+			if ( $api_secret_length > 0 &&
```

Comment:
The length check is unnecessary here since the validator should be doing it.

Reply:
Same as above.